tests/test_sync.py

```diff
@@ -0,0 +1,82 @@
+from time import sleep
+
+from plexapi.sync import Policy, MediaSettings, VIDEO_QUALITY_3_MBPS_720p
+
+
+MAX_RETRIES = 3
+
+
+def get_new_synclist(account, client_id, old_sync_items=[]):
```

Comment:
old_sync_items is mutable, is this intended?

Reply:
I've already rewrote this part locally, anyway :D